src/Compute/Compute/Generated/ResourceSku/ResourceSkuListMethod.cs

```diff
@@ -44,6 +44,18 @@ public partial class GetAzureRmComputeResourceSku : ComputeAutomationBaseCmdlet
         [LocationCompleter("Microsoft.Compute/locations/publishers")]
         public string Location { get; set; }
 
+        [Parameter(
+          Mandatory = false,
+          ValueFromPipelineByPropertyName = true,
+          HelpMessage = "Sets the edge zone name. If set, the query will be routed to the specified edgezone instead of the main region.")]
+        public string EdgeZone { get; set; }
+
+        [Parameter(
+          Mandatory = false,
+          ValueFromPipelineByPropertyName = true,
+          HelpMessage = "Sets and Gets Resource Type")]
+        public string ResourceType { get; set; }
```

Comment:
`ResourceType` is added but never used. And the document also not updated about it.